pkg/commands/drift/statefiles/statefiles.go

```diff
@@ -170,9 +173,16 @@ func (c *DriftStatefilesCommand) Run(ctx context.Context, args []string) error {
 	if err != nil {
 		return fmt.Errorf("failed to absolute path for directory: %w", err)
 	}
-	c.directory = dirAbs
 
-	c.gitClient = git.NewGitClient(c.directory)
+	tmpDir, err := os.MkdirTemp(dirAbs, "tmp")
+	if err != nil {
+		log.Fatal(err)
```

Comment:
Why log.Fatal instead of returning error?

Reply:
Whoops. Copy and paste from godocs mistake.